package.json

```diff
@@ -50,7 +50,7 @@
 		"@types/cookie-parser": "^1.4.2",
 		"@types/debug": "^4.1.7",
 		"@types/express-jwt": "^6.0.4",
-		"@types/ioredis": "^4.28.10",
+		"@types/ioredis": "^5.0.0",
```

Comment:
According to the ioredis changelog and upgrade guide they say that the types are included now. You should not have a @types library here.